### ISSUE TITLE
Sidekick Dead? Can't download it

### ISSUE THREAD

**Issue by YStrauch** [ProgressNS/sidekick-feedback#489]
### Did you verify whether this issue has already been reported here?
Yes

### Tell us about the problem
The installation docs (https://docs.nativescript.org/angular/sidekick/intro/installation and https://www.youtube.com/watch?v=tG7c2nZvhTg) mention the URL https://nativescript.org/nativescript-sidekick, which redirects to the homepage, which in turn doesn't mention sidekick (neither on the homepage nor on tools/developers). I didn't find any official sidekick download which makes me guess that the project is dead.

### Which platform(s) does the issue occur on?
Website (FF/Chrome)

### Provide the following version numbers that the issue occurs with:
n\a

### Tell us how to recreate the issue in as much detail as possible
1. Visit https://nativescript.org/nativescript-sidekick
2. It redirects to https://nativescript.org/
3. ???

### Does this issue happen every time?
Yes

### Send the client logs
n/a

**Comment by stoskov** [comment]
Hi @YStrauch. 

You can find the Sidekick documentation and download links here https://github.com/ProgressNS/sidekick-docs/wiki/Installation.

Hope this helps.

**Comment by YStrauch** [comment]
Thank you alot! Would be helpful if the official tutorials were udpated though

**Comment by asharghi** [comment]
Does anyone know why it was removed from the site?

**Comment by ghost** [comment]
Yeah, I spent quite some time trying to figure out how to get Sidekick. I have it on one of my machines but not the other and couldn't get a download link. This should really be fixed or the official documentation should be updated on your site.

**Comment by vallemar** [comment]
It was very difficult for me to get to the download but what surprised me was the quality of the program.

How can this be so hidden?
I would put it on the cover of NativeScript!
